staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go

```diff
@@ -269,6 +269,17 @@ type VolumeStats struct {
 	// Reference to the PVC, if one exists
 	// +optional
 	PVCRef *PVCReference `json:"pvcRef,omitempty"`
+
+	// VolumeHealthStats contains data about volume health
+	// +optional
+	VolumeHealthStats *VolumeHealthStats `json:"volumeHealthStats,omitempty"`
```

Comment:
Does this need to be included in the summary API?  In general, the summary API contains metrics about resource usage, whereas the kubelet's /metrics endpoint contains the kubelet's operational metrics (e.g. health, latency, etc).  Based on usage i'm aware of, people tend to scrape either the summary API or /metrics/cadvisor, and then also scrape /metrics as well.  For users that do this, they would end up with duplicate metrics.

Reply:
Hi @dashpole, let me know if I missed anything.
I see that Summary API contains PodStats:
https://github.com/kubernetes/kubernetes/blob/v1.24.0-alpha.3/staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go#L28
PodStats contains VolumeStats:
https://github.com/kubernetes/kubernetes/blob/v1.24.0-alpha.3/staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go#L127
VolumeHealthStats is added in VolumeStats here. Should this be sufficient?

Reply:
I'm saying you should probably add these metrics to either `/metrics` or the summary API, but probably not both

Reply:
I see... looks we've already done that for a bunch of volume metrics.